### ISSUE TITLE
Workspace- Error message isn't displayed when edit a Workspace name to <b></b>

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#12966]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Issue found when executing PR** https://github.com/Expensify/App/pull/11465

## Action Performed:
1. Launch the app
2. Log in with any account
3. Go to Setting - Workspace
4. Edit a Workspace name to <b></b> and click Save.

## Expected Result:
The error is shown as Invalid Workspace name

## Actual Result:
Error message isn't displayed when edit a Workspace name to <b></b>

## Workaround:
Unknown

## Platform:

Where is this issue occurring?

- Web
- iOS
- Android
- Desktop App
- Mobile Web

**Version Number:** 1.2.30.0

**Reproducible in staging?:** Yes

**Reproducible in production?:** Yes

**Email or phone of affected tester (no customers):** any

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** Any additional supporting documentation

https://user-images.githubusercontent.com/93399543/203631635-7e7df205-0fb2-42c8-91f8-6d90061dd2b9.MP4

**Expensify/Expensify Issue URL:** 

**Issue reported by:** Applause - Internal Team

**Slack conversation:** 

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by Puneet-here** [comment]
cc: @youssef-lr

**Comment by luacmartins** [comment]
Gonna pick this one up

**Comment by youssef-lr** [comment]
This PR hasn't been merged yet which should fix this. https://github.com/Expensify/Web-Expensify/pull/35548.

**Comment by youssef-lr** [comment]
Also this is a duplicate of https://github.com/Expensify/App/issues/12268, we will need to add validation in the frontend as well.

**Comment by luacmartins** [comment]
Thanks for linking that issue @youssef-lr! I added this issue to the list of fixed issues on the PR you linked. I'll mark this as in review and we can retest once the Web-E PR is merged!

**Comment by ctkochan22** [comment]
Should we just close this in favor of https://github.com/Expensify/App/issues/12268 ? Essentially the same bug, just worded differently I think

**Comment by luacmartins** [comment]
Sounds good to me. Closing in favor of https://github.com/Expensify/App/issues/12268. Thanks for the input!
